packages/actor-rdf-source-identifier-hypermedia-tree/components/Actor/RdfSourceIdentifier/HypermediaQpf.jsonld

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
I'm confused what this file is doing here. An artifact of older code?

Reply:
This is indeed an artifact of older code. Deleted it.

Reply:
This is indeed an artifact of older code. I deleted it.